File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ThirdPartyNetworkShortResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ThirdPartyNetworkShortResponse 
{
    public long Updated { get; set; }
    public long Created { get; set; }
    public bool Active { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string NetworkUID { get; set; }
    
    /// <summary>
    /// Gets or Sets Network
    /// </summary>
    public enum NetworkEnum
    {
        
        /// <summary>
        /// Enum FACEBOOKEnum for FACEBOOK
        /// </summary>
        FACEBOOKEnum = 1,
        
        /// <summary>
        /// Enum TWITTEREnum for TWITTER
        /// </summary>
        TWITTEREnum = 2,
        
        /// <summary>
        /// Enum CLEAREnum for CLEAR
        /// </summary>
        CLEAREnum = 3,
        
        /// <summary>
        /// Enum USERNAMEEnum for USERNAME
        /// </summary>
        USERNAMEEnum = 4,
        
        /// <summary>
        /// Enum PHONEEnum for PHONE
        /// </summary>
        PHONEEnum = 5,
        
        /// <summary>
        /// Enum UNKNOWNEnum for UNKNOWN
        /// </summary>
        UNKNOWNEnum = 6,
        
        /// <summary>
        /// Enum CUSTOMEnum for CUSTOM
        /// </summary>
        CUSTOMEnum = 7
    }

    public NetworkEnum Network { get; set; }
}

File: server-scala-play-server/app/model/ThirdPartyNetworkShortResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ThirdPartyNetworkShortResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ThirdPartyNetworkShortResponse(
  updated: Option[Long],
  created: Option[Long],
  active: Option[Boolean],
  name: Option[String],
  description: Option[String],
  networkUID: Option[String],
  network: Option[ThirdPartyNetworkShortResponse.Network.Value]
)

object ThirdPartyNetworkShortResponse {
  implicit lazy val thirdPartyNetworkShortResponseJsonFormat: Format[ThirdPartyNetworkShortResponse] = Json.format[ThirdPartyNetworkShortResponse]

  // noinspection TypeAnnotation
  object Network extends Enumeration {
    val FACEBOOK = Value("FACEBOOK")
    val TWITTER = Value("TWITTER")
    val CLEAR = Value("CLEAR")
    val USERNAME = Value("USERNAME")
    val PHONE = Value("PHONE")
    val UNKNOWN = Value("UNKNOWN")
    val CUSTOM = Value("CUSTOM")

    type Network = Value
    implicit lazy val NetworkJsonFormat: Format[Value] = Format(Reads.enumNameReads(this), Writes.enumNameWrites[this.type])
  }
}

